cmd/open.js

```diff
@@ -40,7 +31,11 @@ exports.handler = (argv) => {
 };
 
 function getService(argv, res) {
-    if (argv.npms) { return `https://npms.io/search?term=${argv.package}`; }
-    if (argv.npm) { return res.body.collected.metadata.links.npm; }
-    return res.body.collected.metadata.links.repository || res.body.collected.metadata.links.npm;
+    const links = res.body.collected.metadata.links;
+
+    if (argv.link) {
```

Comment:
I used this approach so that it would be backwards compatible with existing version.  

If user does NOT use the --link option, it will still work as before

```
$ npms open gulp
```

is same as 

```
$ npms open --link auto gulp
```

If the `--link` option is not supplied, it will fall through to line 45 which is same as current version.